lib/iis_shc_issuer/v2_to_fhir.rb

```diff
@@ -9,10 +9,13 @@ module V2ToFHIR
     # @param v2_response [HL7::Message] V2 message returned from the IIS Sandbox
     # @return [String] FHIR Bundle representation of the V2 message
     def translate_to_fhir(v2_response)
-      fhir_response = Faraday.post("#{Rails.application.config.v2_to_fhir_host}/api/v0.1.0/convert/text",
+      fhir_response = Faraday.post("#{Rails.application.config.v2_to_fhir_host}/convert/text",
                                    v2_response.to_hl7,
                                    'Content-Type' => 'text/plain')
-      fhir_response.body
+      response_body = fhir_response.body
+      raise IISSHCIssuer::V2ParsingError, JSON.parse(response_body)['errors'] if JSON.parse(response_body)['errors']
```

Comment:
I might check fhir_response.status here and raise an error if it's not 2xx, regardless of whether the error messages are populated. Only errors from the API have the errors message body. For the wrong API endpoint, you get a generic 404 with a text/html body.

Reply:
Good point. Fixed with latest commit.